addons/sys_gui/fnc_initDisplayInventory.sqf

```diff
@@ -46,3 +46,53 @@ _ctrlContainerItems ctrlAddEventHandler ["MouseButtonUp", {
     // Make sure any moved ItemRadio is actually fully moved before resetting hold off
     [FUNC(inventoryListMouseUp), _this, 1] call CBA_fnc_waitAndExecute;
 }];
+
+// Show channel/frequency information for radios
+private _fnc_update = {
+    params ["_display"];
+
+    private _pttAssign = [] call EFUNC(api,getMultiPushToTalkAssignment);
+
+    {
+        private _ctrl = _display displayCtrl _x;
+
+        for "_i" from 0 to (lbSize _ctrl - 1) do {
+            private _item = _ctrl lbData _i;
+
+            if (_item call EFUNC(api,isRadio)) then {
+                private _name = _item call EFUNC(api,getDisplayName);
+
+                // Display frequency for single-channel radios (e.g. AN/PRC-77)
+                private _maxChannels = [_item, "getState", "channels"] call EFUNC(sys_data,dataEvent);
+
+                private _text = if (isNil "_maxChannels") then {
+                    private _txData = [_item, "getCurrentChannelData"] call EFUNC(sys_data,dataEvent);
+                    private _currentFreq = HASH_GET(_txData, "frequencyTX");
+                    format ["%1 %2 MHz", _name, _currentFreq];
+                } else {
+                    format [LELSTRING(ace_interact,channelShort), _name, _item call EFUNC(api,getRadioChannel)]
+                };
```

Comment:
We could put that in a core function as both ACE Interact and this uses the exact same logic.